make-util.js

```diff
@@ -242,7 +245,12 @@ var copyGroup = function (group, sourceRoot, destRoot) {
     mkdir('-p', dest);
 
     // copy the files
-    cp(source, dest);
+    if (group.hasOwnProperty('options') && group.options) {
+        cp(group.options, source, dest);
+    }
+    else {
+        cp(source, dest);
```

Comment:
i was going to make use of this but then didn't. i'm planning to just leave it in for now anyway. i was trying to solve the copy-resources problem for non-node tasks (and modules) in an elegant way. i ended up backing out my changes because it was hideous.